packages/fast-jest-snapshots-react/README.md

```diff
@@ -30,3 +30,28 @@ const testSuite: ISnapshotTestSuite<IMyReactComponentProps> = {
 // Generate snapshots using the prop and component data you defined
 generateSnapshots(testSuite);
 ```
+
+### Snapshot descriptions
+Descriptions can also optionally be applied to any snapshot. This is useful for providing context to the test and what expectation of the snapshot is.

```

Comment:
suggest this "Descriptions can optionally be applied to any snapshot for providing context to the test and the expectation of the test."